tests/runtest.sh

```diff
@@ -918,6 +919,11 @@ fi
 finish_remaining_tests
 
 print_results
+
+time_end=$(date +"%s")
+time_diff=$(($time_end-$time_start))
+echo "$(($time_diff / 60)) minutes and $(($time_diff % 60)) seconds elapsed for unit test."
```

Comment:
Can you change "seconds elapsed for unit test." to "seconds taken to run CoreCLR tests."